fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/jbatch/bulkdata/load/ChunkWriter.java

```diff
@@ -286,4 +263,33 @@ private void pushImportOperationOutcomes2COS(ImportTransientUserData chunkData)
             chunkData.getBufferStreamForImportError().reset();
         }
     }
+    
+    @Override
+    public void open(Serializable checkpoint) throws Exception {
+        if (fhirValidation != null) {
+            isValidationOn = fhirValidation.equalsIgnoreCase("Y");
+        }
+        if (fhirTenant == null) {
+            fhirTenant = "default";
+            logger.info("open: Set tenant to default!");
+        }
+        if (fhirDatastoreId == null) {
+            fhirDatastoreId = Constants.DEFAULT_FHIR_TENANT;
+            logger.info("open: Set DatastoreId to default!");
+        }
+
+        FHIRRequestContext.set(new FHIRRequestContext(fhirTenant, fhirDatastoreId));
+        boolean isCosClientUseFhirServerTrustStore = FHIRConfigHelper
+            .getBooleanProperty(FHIRConfiguration.PROPERTY_BULKDATA_BATCHJOB_USEFHIRSERVERTRUSTSTORE, false);
```

Comment:
I'd prefer to make the default true

Reply:
as mentioned above, this will break S3 and IBM COS services which use public CAs instead of self signed certs.